alchemy-web/docs/getting-started.md

```diff
@@ -1,40 +1,100 @@
 ---
 title: Getting Started with Alchemy
-description: Quick start guide to using Alchemy, the TypeScript-native Infrastructure-as-Code library. Learn how to install, create resources, and manage their lifecycle.
+description: Quick start guide to using Alchemy, the TypeScript-native Infrastructure-as-Code library. Deploy your first Cloudflare Worker with type-safe infrastructure code.
 ---
 
 # Getting Started with Alchemy
 
+This guide walks you through deploying your first Cloudflare Worker using Alchemy, demonstrating Infrastructure-as-Code principles with a real-world, practical example.
```

Comment:
Could be simpler and more concise. "This guide will deploy a Cloudflare worker and bind resources to it with alchemy" or something like that